inmanta.spec

```diff
@@ -4,8 +4,10 @@
 # * version: Version of inmanta-service-orchestrator release (without build_tag)
 # * buildid: Build_tag inmanta-oss RPM
 # * buildid_egg: Build_tag inmanta pypi package
-# * inmanta_dashboard_version: Fully qualified version inmanta-dashboard NPM packge (version number + build_tag)
-# * inmanta_core_version: Fully qualified version inmanta-core pypi packge (version number + build_tag)
+# * inmanta_core_version: Fully qualified version inmanta-core pypi package (version number + build_tag)
+# * inmanta_ui_version: Fully qualified version inmanta-ui pypi package (version number + build_tag)
```

Comment:
Why are these two parameters required?

Reply:
I think we can drop them. They are present in the iso product's spec file as well but I think they are just an artifact from when we built separate RPMs. Iirc the components are now just fetched from the Python environment. I'll get rid of these two lines.